### PR TITLE
gdal source: handle netcdf subdataset and log

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

File: src/util/gdal_timesnap.cpp [hunk 1]
```diff
@@ -110,13 +112,19 @@ GDALTimesnap::GDALDataLoadingInfo GDALTimesnap::getDataLoadingInfo(Json::Value d
...
+    Log::debug(concat("getDataLoadingInfo: time_format: ", time_format));
 	std::string time_start 	= channelJson.get("time_start", datasetJson.get("time_start", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: time_start: ", time_start));
 	std::string time_end 	= channelJson.get("time_end", datasetJson.get("time_end", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: time_end: ", time_end));
 
     std::string path 	 = channelJson.get("path", datasetJson.get("path", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: path: ", path));
     std::string fileName = channelJson.get("file_name", datasetJson.get("file_name", "")).asString();
+    Log::debug(concat("getDataLoadingInfo: fileName: ", fileName));
```
Comment (ChristianBeilschmidt) [inline src/util/gdal_timesnap.cpp]: I guess you should reduce the log messages to a minimum. This spams the log.
For all introduced log messages: Are they really necessary or was it for debuggin a specific problem?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
The netcdf addition is reasonable, but you should look at the logging part.

**Comment by jdroenner** [comment]
well they are debug logs and help to figure out why a file is not loaded. They only appear if you switch logging to debug. then you want to see them...

File: src/util/gdal_timesnap.cpp (hunk 1 above)
Comment (jdroenner) [reply src/util/gdal_timesnap.cpp]: well they are debug logs and help to figure out why a file is not loaded. They only appear if you switch logging to debug. then you want to see them...
I add them everytime something is wrong with the gdalsource.

**Comment by jdroenner** [comment]
please re-review

**Pull request merged**